### DISCUSSION TITLE
shortened links not embedding in Discord

### DISCUSSION THREAD

**Discussion by Dyvinia** [YOURLS/YOURLS#3097] [Q&A]
I'm wondering why my shortened links are not embedding on discord, I know that my squarespace site (which i am planning to migrate) allowed embedding of redirected links (https://dyvine57.com/youtube for example embedded on discord as if I posted the result link). I also saw that bit.ly shortened links worked, however my yourls shortened links are not working. 

Any assistance would be much appreciated
Link: https://dyy.vin
You can create your own shortlink on the main webpage to test for yourself

**Comment by ozh** [comment]
I don't get what "a link is not embedding on discord" means

**Comment by matthewpkemp** [comment]
My interpretation is this is a duplicate of previous issues such as [#2995](https://github.com/YOURLS/YOURLS/discussions/2995). Discord looks for certain [meta tags](https://stackoverflow.com/questions/54266598/meta-tags-for-url-with-description-and-image-in-skype-discord) which I do not believe it will follow a redirect for.

**Reply by Dyvinia** [reply]
it seems that https://u.nu/ which uses yourls also embeds within discord, do you know how that could be achieved
